core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -258,4 +275,21 @@ class SslAdminIntegrationTest extends SaslSslAdminIntegrationTest {
     assertTrue(metrics.nonEmpty, s"Unable to find metric $name: allMetrics: ${allMetrics.keySet.map(_.getMBeanName)}")
     metrics.map(_.asInstanceOf[Gauge[Int]].value).sum
   }
+
+  override def createAdminClient: Admin = {
+    val config = new Properties()
+    config.put(AdminClientConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, "40000")
+    config.put(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, "20000")
+    val client = createAdminClient(configOverrides = config)
+    adminClients += client
```

Comment:
The call on the line above already adds the client to `adminClients` from a super class so this is not needed. Same for `adminClients.foreach(_.close())` in`tearDown()`